src/views/components/cart.js

```diff
@@ -100,7 +87,7 @@ export default class Cart extends Component {
                             this.setState({
                               totalCost: this.state.dataList.reduce(
```

Comment:
Follow this blog and check how to use state values inside setState method.
[Dev Blog](https://dev.to/nicolelc16/react-previous-state-4akk#:~:text=state%20in%20setState()%20.,on%20what%20it%20was%20before.)